features/admin.sms-templates.v1/api/use-get-sms-template.ts

```diff
@@ -51,9 +52,15 @@ const useGetSmsTemplate = <Data = SMSTemplate, Error = RequestErrorInterface>(
         method: HttpMethods.GET,
         url:
             store.getState().config.endpoints.smsTemplates +
-            `/template-types/${templateType}/org-templates/${smsLocale}?resolve=true`
+            `/template-types/${templateType}/org-templates/${smsLocale}`
     };
 
+    if (fetchInheritedTemplate) {
+        requestConfig.url =
+                store.getState().config.endpoints.smsTemplates +
+                `/template-types/${templateType}/org-templates/${smsLocale}?resolve=true`
+    }
```

Comment:
We can specify `params` property in the requestConfig to pass query parameters, instead of attaching manually to the URL.

```suggestion
    if (fetchInheritedTemplate) {
        requestConfig.params = {
        	resolve: true
        }
    }
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7345/commits/842c3356a4eca454a317e9ef60fa2567980034de